README.md

```diff
@@ -68,9 +75,9 @@ Stack is setup to launch stack in the us-west-2 (Oregon) region in your account:
 
 1. Push your container to a registry somewhere (e.g., [Docker Hub](https://hub.docker.com/), [Amazon ECR](https://aws.amazon.com/ecr/)).
 2. Copy one of the existing service templates in [services/*](/services).
-3. Update the `ContainerName` and `Image` parameters to point to your container image instead of the example container.
+3. Update the `ContainerName` and `Image` parameters to point to your container image instead of the example container. Also make sure the `ContainerName` of property `LoadBalancers` of the ECS service matches with the `Name` in `ContainerDefinitions` property of `TaskDefinition`. 
```

Comment:
Is it also necessary to change the `Image` value (at least the final suffix) found under service.yaml > Resources > TaskDefinition > Properties > ContainerDefinitions ?